Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -347,6 +350,8 @@ namespace Multiplayer
 
     void MultiplayerSystemComponent::OnTick(float deltaTime, [[maybe_unused]] AZ::ScriptTimePoint time)
     {
+        AZ_PROFILE_SCOPE(MULTIPLAYER, "MultiplayerTick");
```

Comment:
This is already in the scope MULTIPLAYER but probably needs a tighter name so its clear. I would suggest "MultiplayerSystemComponent::OnTick" or "Component::OnTick"

Reply:
The issue is that the budget does not appear in the PIX profiler.
![image](https://user-images.githubusercontent.com/5432499/154529714-4705397a-da1d-43b5-8bdb-47e6977bd678.png)

Reply:
Correct, the budget is only used to denote the event track color for PIX.  The ImGui profiler is currently the only one to include the budget in the event track name but that may change down the line.  So +1 to using descriptive scope names.